src-tauri/src/commands.rs

```diff
@@ -401,17 +413,11 @@ pub async fn get_miner_metrics(
         randomx_network_hashrate,
         block_height,
         block_time,
-        is_synced,
+        initial_sync_achieved,
         block_reward,
+        base_node_state,
+        failed_checkpoints,
     } = node_status;
-    // let (sha_hash_rate, randomx_hash_rate, block_reward, block_height, block_time, is_synced) = state.node_manager
-    //     .get_network_hash_rate_and_block_reward().await
-    //     .unwrap_or_else(|e| {
-    //         if !matches!(e, NodeManagerError::NodeNotStarted) {
-    //             warn!(target: LOG_TARGET, "Error getting network hash rate and block reward: {}", e);
-    //         }
-    //         (0, 0, MicroMinotari(0), 0, 0, false)
-    //     });
```

Comment:
Any reason to keep this? 

```suggestion
```

Reply:
That looks like a bad merge - I removed it